usmqe_tests/api/gluster/conftest.py

```diff
@@ -3,22 +3,13 @@
 from usmqe.gluster import gluster
 import usmqe.inventory as inventory
 
-
-@pytest.fixture
-def valid_cluster_id(valid_session_credentials):
-    """
-    Generate valid id of imported cluster.
-    """
-    # TODO change
-    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
-    cluster_list = api.get_cluster_list()
-    return cluster_list[0]["cluster_id"]
```

Comment:
Fine, I could live with that.
However at least in those tests you are changing in this PR it should be corrected.
Moreover even in new tests (brick tests) you are adding in this PR it is still used. And that is wrong for sure.